job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/triggers/PullRequestBuilderContext.groovy

```diff
@@ -90,4 +99,28 @@ class PullRequestBuilderContext extends AbstractContext {
     void allowMembersOfWhitelistedOrgsAsAdmin(boolean allowMembersOfWhitelistedOrgsAsAdmin = true) {
         this.allowMembersOfWhitelistedOrgsAsAdmin = allowMembersOfWhitelistedOrgsAsAdmin
     }
+
+    /**
+    * @since 1.22
```

Comment:
Fix the since tag, the next release is 1.37. And add the since tag to all new methods.